src/MemoryTracing/MemoryTracingUtils.cpp

```diff
@@ -24,18 +24,12 @@ using orbit_grpc_protos::kMissingInfo;
 using orbit_grpc_protos::ProcessMemoryUsage;
 using orbit_grpc_protos::SystemMemoryUsage;
 
-SystemMemoryUsage ParseMemInfo(std::string_view meminfo_content) {
-  SystemMemoryUsage memory_info;
-  memory_info.set_total_kb(kMissingInfo);
-  memory_info.set_free_kb(kMissingInfo);
-  memory_info.set_available_kb(kMissingInfo);
-  memory_info.set_buffers_kb(kMissingInfo);
-  memory_info.set_cached_kb(kMissingInfo);
-
-  if (meminfo_content.empty()) return memory_info;
+void GetValuesFromMemInfo(std::string_view meminfo_content,
+                          SystemMemoryUsage* system_memory_usage) {
+  if (meminfo_content.empty()) return;
 
   std::vector<std::string> lines = absl::StrSplit(meminfo_content, '\n');
-  if (lines.empty()) return memory_info;
+  if (lines.empty()) return;
```

Comment:
this lines.empty() is not the same case than meminfo_content.empty()?

Reply:
Ah right, I didn't notice this. Thanks for pointing out it! 

I removed some redundant `lines.empty()`.